release/scan_docker_images.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
+
+function scan_docker_images {
+	local api_url="https://api.eu.prismacloud.io"
+	local console="https://europe-west3.cloud.twistlock.com/eu-1614931"
+	local image_names=("$@") # Capture all arguments as an array
+
+	LIFERAY_PRISMA_ACCESS_KEY=""
+	LIFERAY_PRISMA_SECRET=""
+
+	local request_data=$(
+		cat <<- END
+		{
+			"password": "${LIFERAY_PRISMA_SECRET}",
+			"username": "${LIFERAY_PRISMA_ACCESS_KEY}"
+		}
+		END
+	)
+
+	local auth_response=$(\
+		curl \
+			--data "${request_data}" \
+			--header "Content-Type: application/json" \
+			--request POST \
+			--silent \
+			"${api_url}/login")
+
+	local token=$(echo "${auth_response}" | jq -r '.token')
```

Comment:
@lucasmiranda0, we need to check if this `curl` is successful, e.g. if the login is indeed performed. Otherwise, the token will be empty and, in this case, the scan should fail with `LIFERAY_COMMON_EXIT_CODE_BAD`.